src/box/lua/misc.cc

```diff
@@ -350,93 +348,6 @@ lbox_txn_set_isolation(struct lua_State *L)
 	return 1;
 }
 
-/** {{{ Utils to work with tuple_format. **/
-
-struct tuple_format *
-lbox_check_tuple_format(struct lua_State *L, int narg)
-{
-	uint32_t ctypeid;
-	struct tuple_format *format =
-		*(struct tuple_format **)luaL_checkcdata(L, narg, &ctypeid);
-	if (ctypeid != CTID_STRUCT_TUPLE_FORMAT_PTR) {
-		luaL_error(L, "Invalid argument: 'struct tuple_format *' "
-			   "expected, got %s)",
-			   lua_typename(L, lua_type(L, narg)));
-	}
-	return format;
-}
-
-static int
-lbox_tuple_format_gc(struct lua_State *L)
```

Comment:
Please, do all the code relocations in a separate preparatory commit. Otherwise it is impossible to see what was moved as is and what was changed.

Reply:
Added a separate commit.